package.json

```diff
@@ -29,5 +43,17 @@
   "devDependencies": {
     "jest": "^26.0.1",
     "remark": "^13.0.0"
+  },
+  "eslintConfig": {
```

Comment:
[note] Renaming to `.eslintrc.cjs` would be required, so I've moved the config into `package.json`.